src/components/tag/__tests__/Tag.spec.js

```diff
@@ -0,0 +1,65 @@
+import Tag from "../Tag.svelte";
+import { fireEvent } from "@testing-library/svelte";
+import { render, clearContext, delay } from "@/components/_util/testHelpers";
+import { PresetColors } from "@/components/_util/colors.js";
+import { CheckCircleOutlined } from "@/components/icons";
+
+describe("Tag component", () => {
+  afterEach(() => {
+    clearContext();
```

Comment:
If you don't use `$$context` there is no need to call `clearContext()`

Reply:
Done 😄